simapp/app.go

```diff
@@ -407,7 +408,7 @@ func NewSimApp(
 	app.mm.RegisterServices(app.configurator)
 
 	// add test gRPC service for testing gRPC queries in isolation
-	testdata.RegisterQueryServer(app.GRPCQueryRouter(), testdata.QueryImpl{})
+	testdata_pulsar.RegisterQueryServer(app.GRPCQueryRouter(), testdata_pulsar.QueryImpl{})
```

Comment:
Once we release v0.46, people will have a look at this file, and maybe blindly copy-paste this.

It's testdata, so IMO it's fine. Do you see concerns here (e.g. using alpha software)?